components/Forms/NewCollection.js

```diff
@@ -0,0 +1,47 @@
+import updateUserinDb from "@/helpers/updateUserInDb";
+import styled from "styled-components";
+
+import Plus from "/public/icons/svg/plus.svg";
+import StyledInput from "@/components/Styled/StyledInput";
+import AddButton from "@/components/Styled/AddButton";
+import ModalComponent from "../Modal";
+import StyledH2 from "../Styled/StyledH2";
+
+export default function NewCollection({ user, mutateUser, setModal }) {
+  function addNewCollection(event) {
+    event.preventDefault();
+    const collectionName = event.target.collectionName.value;
+    const newCollection = { collectionName: collectionName, recipes: [] };
+
+    if (!user.collections) {
+      user.collections = [];
+    }
+    user = {
+      ...user,
+      collections: [...user.collections, newCollection],
+    };
+
+    updateUserinDb(user, mutateUser);
+    setModal();
+  }
+  return (
+    <ModalComponent toggleModal={setModal}>
+      <StyledH2>Wie soll dein neues Kochbuch heißen?</StyledH2>
+      <StyledForm onSubmit={addNewCollection}>
+        <StyledInput name="collectionName" placeholder="Kochbuch Name" />
```

Comment:
this input is missing its label

Reply:
I believe my suggestion was not clear enough. I did not mean an aria-label, but the html element <label>.

Every form input should have its own label to be properly accessible.

In case you only want it to be readable by a screenreader, you can style the label to be invisible on the page:

```js
<StyledLabel htmlFor="collectionName">Kochbuch Name</StyledLabel>
 <StyledInput name="collectionName" placeholder="Kochbuch Name" id="collectionName" />

// ...
const StyledLabel = styled.label`
    position: absolute;
    width: 1px;
    height: 1px;
    padding: 0;
    margin: -1px;
    overflow: hidden;
    clip: rect(0, 0, 0, 0);
    border: 0;
`
```